README.md

```diff
@@ -58,11 +58,18 @@ Quickwit is the next-gen search & analytics engine built for log management. It
 - Search stream API that notably unlocks full-text search in ClickHouse
 
 
-### 🔮 Upcoming Features
-- Ingest your logs from your object storage
-- Distributed indexing
-- Support for tracing
-- Native support for OpenTelemetry
+### 🔮 Roadmap
+- [Quickwit 0.4 - August 2022](https://github.com/quickwit-oss/quickwit/projects/5)
+  - Native support for Kubernetes
+  - Boolean, datetime, and IP address fields
+  - Range queries
+  - Sort
+  - Retention policies
+- [Quickwit 0.5 - October 2022](https://github.com/quickwit-oss/quickwit/projects/6)
+  - Grafana data source
+  - Native support for OpenTelemetry
+  - REST API for managing indexes
+- [Long-term roadmap](ROADMAP.md)
```

Comment:
Enhancement: you could add the big three features we will implement on the long term roadmap and add a last item with "and [more](link)"

Reply:
Good suggestion. I went with:
  - Distributed indexing
  - Pipe-based query language
  - Security (TLS, authentication, RBAC)